torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -341,6 +342,37 @@ def get_default_x86_inductor_quantization_config(
     return quantization_config
 
 
+@functools.lru_cache
+def get_x86_inductor_linear_dynamic_fp16_config():
+    """
+    For linear_dynamic_fp16
+    Although it is called dynamic, it is actually static in the sense that input and weight are converted to fp16 directly.
```

Comment:
I think for fp16 static and dynamic might be the same since there is no qparams to choose here.
dynamic: choose_qparams -> q -> dq
static: (qparams chosen before) q -> dq

Reply:
Thanks. I have updated the description here.